src/content/docs/style-guide/images/embed-images.mdx

```diff
@@ -15,61 +15,114 @@ redirects:
 
 We want to ensure that sensitive data does not appear in UI screenshots. That's why we recommend that you provide a permalink to a test account. Do not take a screenshot from your New Relic account.
 
 Here are some things to keep in mind when you're creating an image:
 
-* Make sure your image provides useful information at a glance.
-* Include a caption with helpful context for the image.
-* For screenshot captions, include the path in bold.
-* For video captions, include the approximate running time.
+- Make sure your image provides useful information at a glance.
+- Use alphanumeric characters in combo with `-` or `_` in the image file name. Avoid using any other special characters.
+- Include a caption with helpful context for the image.
+- For screenshot captions, include the path in bold.
+- For video captions, include the approximate running time.
 
-Our doc site images are stored in individual `images` directories at the root level of each taxonomy category. These `images` directories contain all the images used in the docs for that category. Do not link to an external file, all images used in docs must be stored in [our GitHub repo](https://github.com/newrelic/docs-website).
+Do not link to an external file, all images used in docs must be stored in [our GitHub repo](https://github.com/newrelic/docs-website).
+
+<Callout variant="important">
+  Images are managed a little differently between `.mdx` files (content in `src/content/docs`) and `.md`files  (content in `src/content/whats-new`).
+
+  For `.mdx` files, images are stored in the `src/images` directory.
+
+  For `.md` files, images are stored in individual `images` directories at the root level of each taxonomy category. These `images` directories contain all the images used in the docs for that category. 
+</Callout>
 
 To add an image from scratch:
 
-1. Find the images folder for your doc. For example, if your doc lives in the `docs/style-guide/processes-procedures` directory, use the `docs/style-guide/processes-procedures/images` folder.
-2. If the `images` directory does not exist in the directory where your doc lives, use [VScode](https://code.visualstudio.com/docs/getstarted/tips-and-tricks#_create-or-open-a-file) or Finder to create a folder in that directory named `images` (must be lower case and plural).
-3. Place your image in the `images` directory. Give the image a descriptive file name; for example, `fso-ui-overview.png` is much better than `123-go-dawgs.png`. 
-4. Continue with the following procedure to embed the image in a doc.
+1. Place your image in the relevant directory. Give the image a descriptive file name; for example, `fso-ui-overview.png` is much better than `123-go-dawgs.png`.
+
+2. Continue with the following procedure to embed the image in a doc.
 
 ## Embed an image [#embed]
 
-Use markdown to embed an image in a doc. The basic structure:
+<CollapserGroup>
+  <Collapser title={<>Embed in <InlineCode>.mdx</InlineCode> file</>}>
```

Comment:
Not sure what is happening here, but in the readme preview on github its showing "Embed in .mdx file}>"

Reply:
looks as expected when running locally

![2022-03-17_15-43-44](https://user-images.githubusercontent.com/2952843/158906287-8a992a40-488f-43e7-a48b-3cec8ed31788.png)